contrib/world_builder/include/visualization/main.h

```diff
@@ -0,0 +1,32 @@
+/*
+  Copyright (C) 2020 by the authors of the World Builder code.
+
+  This file is part of the World Builder.
+
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU Lesser General Public License as published
+   by the Free Software Foundation, either version 2 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU Lesser General Public License for more details.
+
+   You should have received a copy of the GNU Lesser General Public License
+   along with this program.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+void project_on_sphere(double, double &, double &, double &);
```

Comment:
do you not use header guards?

Reply:
I normally do. Since this is a very simple program contained to one file, it didn't have a headerfile to begin with. When I made a headerfile I apparently forgotten to add header guards. This won't influence aspect in any way, since aspect currently doesn't use this file (it is only used when the project itself is compiled in the directory, aspect currently just uses the core library files directly. In the future I would like to make it a bit more integrated through cmake). I will fix it in the next version of the world builder.